modules/balrog_scriptworker/manifests/init.pp

```diff
@@ -35,17 +34,20 @@
             user            => $users::builder::username,
             group           => $users::builder::group,
             mode            => '0700',
-            packages        => file("balrog_scriptworker/requirements-3.txt");
+            packages        => file("balrog_scriptworker/requirements.txt");
     }
 
+    # TODO: remove once https://github.com/mozilla-releng/build-puppet/pull/520 is landed and ensure => absent has been
+    # propogated to all balrog workers
     python27::virtualenv {
         "${balrog_scriptworker::settings::root}/py27venv":
             python          => $packages::mozilla::python27::python,
             rebuild_trigger => Class['packages::mozilla::python27'],
             require         => Class['packages::mozilla::python27'],
             user            => $users::builder::username,
             group           => $users::builder::group,
-            packages        => file("balrog_scriptworker/requirements-27.txt");
+            ensure          => 'absent',
+            packages        => file("balrog_scriptworker/requirements.txt");
```

Comment:
Nit: I just noticed this line may cause a conflict once deployed

Reply:
Ah, and leave `requirements-27` populated, but note that it's unused? :+1:

Reply:
Hmmm, good call. It's probably not worth it. Let's keep it the way it is now.